2022/ssm/shared-statistical-methods.md

```diff
@@ -0,0 +1,31 @@
+# Shared Statistical Methods
+
+#### Background
+
+Over the years, initiatives like __CSPA__ (UNECE), __ESSNet Serv2__, __ESSnet I3S__ have all tried to solve the problem of sharing statistical services across countries. Sharing is hard, and there are many hurdles to be crossed. Many of those hurdles are technical, organizational and cultural. 
```

Comment:
The actual title for "Serv2" was SCFE (Sharing common functionalities in the ESS)